CMakeLists.txt

```diff
@@ -118,6 +118,11 @@ include(GNUInstallDirs)
 include(autocmake_safeguards)
 include(custom_color_messages)
 
+if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+    set(CMAKE_INSTALL_PREFIX "/usr/local/psi4" CACHE PATH "Install path" FORCE)
```

Comment:
Because psi3 and psi4 have always defaulted to `/usr/local/psi4`, with `/bin`, `/include`, etc. underneath, and I supposed we wanted to continue that.